### DISCUSSION TITLE
How to contribute with new apps

### DISCUSSION THREAD

**Discussion by jms-du-se** [EUCPilots/evergreen-module#586] [Q&A] [answered]
Hi!

I have been searching for information about how I can contribute with additional apps to be included i Evergreen, but can't find it.
Is this information available somewhere?

//Jasper

**Comment by aaronparker** [comment, marked as answer]
I haven't written something specifically on how to contribute new apps - a code and style guide would certainly be useful. There is the documentation here: https://stealthpuppy.com/evergreen/.

At a high level:
* You can add a new app by forking the repo, update code, and create a PR
* Review the `Apps` and `Manifests` directories to see sample apps
* Apps should use the various methods to determine updates - parsing HTML to find downloads won't be accepted 
* Commits must be signed

**Reply by jms-du-se** [reply]
Thank you! I'll see if I can find this out. If I don't, can I suggest applications to be added? (I'll promise I'll try first ;) )

//Jasper
